scripts/build.js

```diff
@@ -15,10 +15,10 @@ var lineRegex = /(<%= settings\.load %>|<%= settings\.page %>|}}\(\);)/g;
 var versionRegex = /<%= settings\.version %>/;
 var snippet = template(source.replace(loadRegex, '$1').replace(pageRegex, '$1').replace(versionRegex, packageJSON.version), { variable: 'settings' });
 var snippetMin = template(minify(source, {
-  mangle: { except: ['analytics'] },
+  mangle: { except: ['analytics', 'key'] },
```

Comment:
Need to exempt `key` because minify happens _before_ `ajsPath` is processed, and `ajsPath` uses the `key` variable.